src/Command/InfectionCommand.php

```diff
@@ -285,6 +286,11 @@ private function startUp(): TestFrameworkAdapter
 
         $this->includeUserBootstrap($config);
 
+        /** @var Filesystem $fileSystem */
+        $fileSystem = $this->container['filesystem'];
+
+        $fileSystem->mkdir($config->getTmpDir());
```

Comment:
do we need `, 0777` as a second parameter (as it was in the `TmpDirCreator`)?

Reply:
we can put it but it's the default so not a necessity